src/libsystemd/sd-journal/journal-file.c

```diff
@@ -4344,6 +4344,11 @@ int journal_file_archive(JournalFile *f, char **ret_previous_path) {
         if (!endswith(f->path, ".journal"))
                 return -EINVAL;
 
+        if (le64toh(f->header->head_entry_seqnum) == 0)
```

Comment:
hmm, i think a check of `f->header->n_objects` being zero is a lot more obvious, no?